tests/model_registry/python_client/test_model_registry_creation.py

```diff
@@ -86,7 +87,17 @@ def test_model_registry_operator_env(
         if not namespace_env:
             pytest.fail("Missing environment variable REGISTRIES_NAMESPACE")
 
-    # TODO: Edit a registered model
-    # TODO: Add additional versions for a model
-    # TODO: List all available models
-    # TODO: List all versions of a model
+    def test_model_registry_grpc_container_removal(self, model_registry_deployment_containers: dict[str, Any]):
+        # RHOAIENG-26239
+        for container in model_registry_deployment_containers:
```

Comment:
may be 
try 
```grpc_container = next((c for c in model_registry_deployment_containers if "grpc" in c["name"]), None)
assert grpc_container is None, f"GRPC container found: {grpc_container}"
```

Reply:
I would need to save grpc_container, I don't see any extra benefit.